### PR TITLE
Add creation of custom bootable install media

### PR DESCRIPTION
**Pull request by dgiebert**
Users will be able to create custom images to customize the boot process; this includes:
- Custom or extra modules that are needed during boot.
- Embed custom configuration 
- Any other patching of the initially booted operating system 

This will also make it easier to roll out custom builds in air-gapped scenarios.

### CODE REVIEW

File: docs/customizing.md [hunk 1]
```diff
@@ -285,7 +285,7 @@ show case how this could be added:
 
 ```docker showLineNumbers
 # The version of Elemental to modify
-FROM registry.opensuse.org/isv/rancher/elemental/teal52/15.3/rancher/elemental-node-image/5.2:VERSION
+FROM registry.opensuse.org/isv/rancher/elemental/teal53/15.4/rancher/elemental-node-image/5.3:VERSION
```
Comment (kkaempf) [inline docs/customizing.md:288]:
```suggestion
FROM registry.opensuse.org/isv/rancher/elemental/stable/teal53/15.4/rancher/elemental-node-image/5.3:VERSION
```

File: docs/customizing.md [hunk 2]
```diff
@@ -318,3 +318,38 @@ be run and verified using docker with
...
 docker run -it myrepo/custom-build:v1.1.1 bash
 ```
+
+## Create a custom bootable installation media
+
+Elemental Teal leverages container images to build its root filesystems; therefore, it is possible
+to use it in a multi-stage environment to create custom bootable media that bundles a custom container image.
+
+```docker showLineNumbers
+FROM registry.opensuse.org/isv/rancher/elemental/teal53/15.4/rancher/elemental-node-image/5.3:latest as os
```
Comment (kkaempf) [inline docs/customizing.md:328]:
```suggestion
FROM registry.opensuse.org/isv/rancher/elemental/stable/teal53/15.4/rancher/elemental-node-image/5.3:latest as os
```

**Review by kkaempf — CHANGES_REQUESTED** [review]
Use images from `stable` only

**Comment by kkaempf** [comment]
It's not immediately clear to me what this PR accomplishes.

Can you please elaborate more on the motivation and the problem you're solving here ?

**Comment by kkaempf** [comment]
Ping @dgiebert 😉

**Comment by dgiebert** [comment]
Users will be able to create custom images to customize the boot process; this includes:
- Custom or extra modules that are needed during boot.
- Embed custom configuration 
- Any other patching of the initially booted operating system 

This will also make it easier to roll out custom builds in air-gapped scenarios.

**Review by kkaempf — APPROVED** [review]

**Pull request merged**